lib/modules/platform/github/index.ts

```diff
@@ -1214,7 +1214,7 @@ async function getIssues(): Promise<Issue[]> {
       variables: {
         owner: config.repositoryOwner,
         name: config.repositoryName,
-        user: config.renovateUsername,
+        ...(!config.ignorePrAuthor && { user: config.renovateUsername }),
```

Comment:
Isn't user a mandatory field of this graphql query?

Reply:
Tested?

Reply:
Yes and it passed :)

But actually the test was faulty because on re-test I found that the `ignorePrAuthor` value I set in my `renovate.json` was not being used. Instead the default `false` was still being passed to the `getIssues` fn, thus the `user` was always being set.

I will draft it for now and check why the correct `ignorePrAuthor` is not being passed to the `getIssues` fn. I doubt that gitlab might have the same issue.

Reply:
Is `ignorePrAuthor` supposed to be self hosted config option?

Reply:
Tested again locally ☑️